pkg/api-server/dataplane_overview_ws.go

```diff
@@ -15,26 +15,28 @@ import (
 	rest_errors "github.com/Kong/kuma/pkg/core/rest/errors"
 )
 
-type overviewWs struct {
+type dataplaneOverviewEndpoints struct {
```

Comment:
Ws is WebService from go-restful. It feel like endpoints is better name here since we add endpoints to existing webservice.

Reply:
I didn't want to rename files to avoid conflicts with community PR as well as for it to be easier to review, but since I've got +1 on this then let me change it.